src/backend/src/middleware/CustomErrorHandler.ts

```diff
@@ -0,0 +1,14 @@
+import { ErrorRequestHandler, Request, Response, NextFunction } from 'express';
```

Comment:
This error handler lets us throw errors from the service, client, controller and sends it back to frontend accordingly.